data_pipeline/gmail_data/get_gmail_data.py

```diff
@@ -85,19 +85,36 @@ def write_dataframe_to_jsonl_file(
         df_data_to_write.to_json(file, orient='records', lines=True, force_ascii=False)
 
 
+# pylint: disable=duplicate-string-formatting-argument
 @backoff.on_exception(backoff.expo, TimeoutError, max_tries=10)
 def get_distinct_values_from_bq(
             project_name: str,
             dataset: str,
             column_name: str,
-            table_name: str
+            table_name_to_execute: str,
```

Comment:
I am slightly confused by the new names. Initially I thought it was a type and was meant to be `...to_exclude`. But then this seems to be the source table whereas the next one is used for exclusion?

Reply:
I am changing the names